### PR TITLE
Week 3 Updates

### PR DESCRIPTION
**Pull request by McJon**
### Goal
Add instructions on how to get a theme ID and how to configure file associations for liquid

### Testing Instructions
- Ensure that these new instructions make sense!
- Wait until after the dry run incase there is something missing

### Reviewers
@natewiebe13

### CODE REVIEW

**Comment by McJon** [comment]
I added another way to get the theme ID. Let me know if this makes sense.

I will be adding the liquid tutorial to this repo as well in another PR.

File: README.md [hunk 1]
```diff
@@ -36,6 +36,15 @@ Full instructions can be found [[here]](https://shopify.github.io/themekit/#conf
...
 Theme Kit can be run using the `./theme` script in the project's root directory. A full list of commands can be found [[here]](https://shopify.github.io/themekit/commands/).
 
+#### 4 Ways to Get Your Theme ID
+1. From the URL
+   - Go to your-store.myshopify.com/admin/themes select "Edit Code" from the "Actions" dropdown next to the theme
+   - The ID should be in the URL of this page
+   - e.g. your-store.myshopify.com/admin/themes/**80643031095**
+2. your-store.myshopify.com/admin/themes.xml
+3. your-store.myshopify.com/admin/themes.json
+4. ./theme get --list -p=[your-password] -s=[you-store.myshopify.com]
```
Comment (natewiebe13) [inline README.md]: Should be:
```suggestion
4. ./theme get --list -p=[your-password] -s=[your-store.myshopify.com]
```

Comment (McJon) [reply README.md]: Ah I thought I fixed that. Thanks.

**Comment by natewiebe13** [comment]
Looks good

**Pull request merged**